src/main/java/ch/bfh/bti7081/view/MainViewContent.java

```diff
@@ -0,0 +1,45 @@
+package ch.bfh.bti7081.view;
+
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+
+import java.util.List;
+
+public class MainViewContent extends VerticalLayout {
+    private String loremipsum = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, " +
+            "sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, " +
+            "sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. " +
+            "Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.";
+
+    public MainViewContent() {
+        // title and welcometext
+        H1 title = new H1("Herzlich Willkommen!");
+        Label welcomeText = new Label(loremipsum);
+        this.add(title);
+        this.add(welcomeText);
+    }
+
+    public void setFeatureView(List<String> seminaries) {
```

Comment:
Je nach dem könnte es besser sein 3 getrennte Funktionen zu haben für das Abfüllen der 3 Feature-Views und dafür die einzelnen Komponenten im Konstruktor zu erstellen.

Reply:
Da das View-Objekt in der Container-Klasse erstellt wird (vor der Presenter-Klasse), sind die Daten bei der Initialisierung noch nicht vorhanden, es muss also später gesetzt werden.
Ob es sich lohnt, drei Methoden zu erstellen kann man noch diskutieren. Aktuell ist es meiner Meinung nach nicht nötig, da die Komponenten verpackt in ein VerticalLayout der View hinzugefügt werden und die beiden FeatureViews nur im Presenter gesetzt werden (und das gleichzeitig)